apps/deploy-web/src/components/alerts/AlertsLayout.tsx

```
@@ -32,9 +32,9 @@ export const AlertsLayout: React.FunctionComponent<Props> = ({ children, page, t
       case AlertTabs.ALERTS:
         router.push(UrlService.alerts());
         break;
-      case AlertTabs.CONTACT_POINTS:
+      case AlertTabs.NOTIFICATION_CHANNELS:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)